src/DataGrid.php

```diff
@@ -258,6 +258,8 @@ class DataGrid extends Control
 
 	protected bool $showSelectedRowsCount = true;
 
+	protected ?string $groupActionsConfirmDialog;
+
```

Comment:
```suggestion
```

Reply:
It's not needed, right?

Reply:
Oh yes, sorry, mistake when I resolved the conflicts with 'next' branch. It's removed now.